lib/ansible/modules/storage/netapp/na_ontap_interface.py

```diff
@@ -260,6 +294,12 @@ def set_options(options, parameters):
             options['administrative-status'] = parameters['admin_status']
         if parameters.get('force_subnet_association') is not None:
             options['force-subnet-association'] = 'true' if parameters['force_subnet_association'] else 'false'
+        if parameters.get('dns_domain_name') is not None:
+            options['dns-domain-name'] = parameters['dns_domain_name']
+        if parameters.get('listen_for_dns_query') is not None:
+            options['listen-for-dns-query'] = str(parameters['listen_for_dns_query'])
```

Comment:
Is it enough to use str, or do we need to convert to lowercase?

Look also at line 296.  Should we use teh same construct?

Reply:
I confirmed the ZAPI accepts both False and false, or True and true, so this is OK as is.

It's only when making comparisons with returned values (get), that we need to be case sensitive.